src/vs/workbench/contrib/webview/browser/baseWebviewElement.ts

```diff
@@ -122,13 +111,6 @@ export abstract class BaseWebview<T extends HTMLElement> extends Disposable {
 			this.handleFocusChange(false);
 		}));
 
-		this._register(this.on('did-keydown', (data: KeyboardEvent) => {
-			// Electron: workaround for https://github.com/electron/electron/issues/14258
```

Comment:
Are you sure the workaround for https://github.com/electron/electron/issues/14258 is no longer required?

Reply:
I tracked and found the shortcuts for webview were responded twice and another was called from [Line 161 in window.ts](https://github.com/microsoft/vscode/blob/230baa73721a5813950e7c4509443d407512f8d4/src/vs/workbench/electron-browser/window.ts#L161)

I reckon that they are of the same purpose and this would be deprecated.